src/main/resources/openapi/annotations.yaml

```diff
@@ -96,3 +163,27 @@ components:
       properties:
         annotation_id:
           type: integer
+
+    AnnotationErrorCode:
+      type: string
+      enum:
+        - "ANNOTATION_100"
+        - "ANNOTATION_101"
+        - "ANNOTATION_102"
+        - "ANNOTATION_103"
+        - "ANNOTATION_104"
+        - "ANNOTATION_105"
+        - "ANNOTATION_106"
+      x-enum-varnames: [ HEARING_NOT_FOUND, FAILED_TO_UPLOAD_ANNOTATION_DOCUMENT, USER_NOT_AUTHORISED_TO_DOWNLOAD, INVALID_ANNOTATIONID_OR_ANNOTATION_DOCUMENTID, INVALID_ANNOTATIONID_OR_ANNOTATION_DOCUMENTID_FOR_JUDGE, FAILED_TO_DOWNLOAD_ANNOTATION_DOCUMENT]
+
+    AnnotationTitleErrors:
+      type: string
+      enum:
+        - "The requested hearing cannot be found"
+        - "The annotation failed to be uploaded"
+        - "user is not authorised to download annotation documents"
+        - "annotation ID or annotation document ID does not exist"
+        - "invalid annotation id or annotation document id"
+        - "invalid annotation id or annotation document id for judge"
+        - "The annotation failed to be downloaded"
+      x-enum-varnames: [ HEARING_NOT_FOUND, FAILED_TO_UPLOAD_ANNOTATION_DOCUMENT, USER_NOT_AUTHORISED_TO_DOWNLOAD, INVALID_ANNOTATIONID_OR_ANNOTATION_DOCUMENTID, INVALID_ANNOTATIONID_OR_ANNOTATION_DOCUMENTID_FOR_JUDGE, FAILED_TO_DOWNLOAD_ANNOTATION_DOCUMENT]
```

Comment:
the enum is not aligned
![image](https://github.com/hmcts/darts-api/assets/131666106/31ead81d-d879-4820-9f0f-628b02f3190e)